File: exercises/tests/morse_test.rs
#[test]
fn sos() {
    assert_eq!(skills::morse::convert_letters("sos"), "...---...");
}

#[test]
fn daily() {
    assert_eq!(skills::morse::convert_letters("daily"), "-...-...-..-.--");
}

#[test]
fn programmer() {
    assert_eq!(
        skills::morse::convert_letters("programmer"),
        ".--..-.-----..-..-----..-."
    );
}

#[test]
fn bits() {
    assert_eq!(skills::morse::convert_letters("bits"), "-.....-...");
}

#[test]
fn three() {
    assert_eq!(skills::morse::convert_letters("three"), "-.....-...");
}

File: exercises/tests/morse.test.js
import test from "./test.js";
import morse from "../src/morse.js";

test({
  description: "morse baseline",
  actual: morse("three"),
  expected: "-.....-...",
});

test({
  description: "morse short",
  actual: morse("sos"),
  expected: "...---...",
});

test({
  description: "morse long",
  actual: morse("programmer"),
  expected: ".--..-.-----..-..-----..-.",
});
